src/main/java/by/dragonsurvivalteam/dragonsurvival/common/capability/DragonStateHandler.java

```diff
@@ -69,6 +69,7 @@ public class DragonStateHandler extends EntityStateHandler {
     public boolean hasUsedAltar;
     public boolean isInAltar;
     public boolean refreshBody;
+    public boolean isJumping = false;
```

Comment:
no need to set this to false since that is the default value

Reply:
I don't like relying on default values, its a C++ thing. But yea it is safe to rely on false in Java.